src/utils/format.ts

```diff
@@ -8,7 +9,7 @@ export const formatTitleCase = (str: string|undefined): string =>  {
 
   return str.replace(
     /\w\S*/g, (word) => {
-      return word.charAt(0).toUpperCase() + word.substr(1).toLowerCase();
+      return word.charAt(0).toUpperCase() + word.substring(1).toLowerCase();
```

Comment:
substr was deprecated, substring is also available as a function on string object